app/controllers/orderController.js

```diff
@@ -9,11 +9,17 @@ class OrderController {
         return ResponseHandler.success(res, orders);
     }
 
-    async buyNewOrder(req, res) {
-        const { strategy_id } = req.query
-        const strategy = await OrderService.buyNewOrder(strategy_id);
+    async buyOrder(req, res) {
+        const { strategy_id } = req.query;
+        const strategy = await OrderService.buyOrder(strategy_id);
```

Comment:
A better naming of the method should be 'place', for instance, OrderService.place(xxx).

The 'order' resides in both service and method name is redundant. There's many best practices teaching how to design the names you could find online. (This is not mandatory, and if it's the style followed by the whole team, it's ok)